src/array.ts

```diff
@@ -77,7 +77,7 @@ function normalizeOffset(value: number, length: number): number {
  * the functionality is required or not.
  */
 export namespace Shim {
-	export function from(arrayLike: Iterable<any> | ArrayLike<any>, mapFunction?: MapCallback<any, any>, thisArg?: any): Array<any> {
+	export function from(this: any, arrayLike: Iterable<any> | ArrayLike<any>, mapFunction?: MapCallback<any, any>, thisArg?: any): Array<any> {
```

Comment:
should this be `ArrayConstructor` right, since below the reference to it is `const Constructor: ArrayConstructor = this;` and could then just be rewritten as `const Constructor = this;`